compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -256,13 +256,13 @@ static LogicalResult setContractConfig(func::FuncOp entryPoint,
 
         SmallVector<unsigned> partitionedLoops =
             cast<PartitionableLoopsInterface>(op.getOperation())
-                .getPartitionableLoops(kNumMaxParallelDims);
+                .getPartitionableLoops(/*maxNumPartitionedLoops=*/5);
         llvm::SmallDenseSet<unsigned, 4> partitionedLoopsSet;
         partitionedLoopsSet.insert(partitionedLoops.begin(),
                                    partitionedLoops.end());
         for (auto loopID : llvm::seq<unsigned>(0, numParallelLoops)) {
           if (!partitionedLoopsSet.count(loopID)) {
-            workgroupTileSizes[loopID] = 1;
+            workgroupTileSizes[loopID] = 0;
```

Comment:
Why is this 0... This should be 1...

Reply:
I reverted the other changes and just did your suggestion for getPartitionableLoops(). I can set them back if thats what you intended. One moment

Reply:
Sorry, I wasn't clear. You need to do both. This needs to be set to 1. I think without that it won't fix the issue